config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presets.yaml

```
@@ -179,3 +179,48 @@ presets:
         path: ca.crt
       - key: vmc-vpn-tls.key
         path: tls.key
+- labels:
+    # For kubernetes-sigs/cluster-api-provider-vsphere janitor
+    preset-vsphere-janitor-config: "true"
```

```suggestion
    preset-cluster-api-provider-vsphere-janitor-config: "true"
```
What about this? Seems to follow the "convention" of using the repo name as a suffix (and the janitor is run from the CAPV repo)

(just seems better namespacing to me)